### PR TITLE
Feat product listing and fillters

### PR DESCRIPTION
**Pull request by ianuragsharma**
Please check the last commit [feat:added product page and filters]
all other commits are either a refactor or have been reviewed .
image

### CODE REVIEW

**Review by ishitakeshawani — APPROVED** [review]
Clean code
Liked that you used barrel export
nice UI

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-      </Routes>
-      <Footer />
+      <div className="body-wrapper">
+        <Navbar />
+        <Routes>
+          <Route path="/" exact element={<HomeScreen />} />
+        </Routes>
+        <Routes>
+          <Route path="/products" exact element={<ProductsScreen />} />
+        </Routes>
```
Comment (hrishikeshmane) [inline src/App.js:15]: You can refactor this to-
<Routes>
          <Route path="/" exact element={<HomeScreen />} />
          <Route path="/products" exact element={<ProductsScreen />} />
</Routes>

File: src/components/filter/Filter.jsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+              id="three"
+              name="Rating"
+              value="3"
+              checked={byRating === "3"}
+              onChange={(event) =>
+                dispatch({ type: "RATING", payload: event.target.value })
+              }
+            />
+            <label htmlFor="three">3 Stars & above</label>
+          </div>
```
Comment (hrishikeshmane) [inline src/components/filter/Filter.jsx:135]: This seem like a repetitive code.  Why not map this through an array of objects

**Review by hrishikeshmane — COMMENTED** [review]
Rest looks good to me.

File: src/App.js (hunk 1 above)
Comment (ianuragsharma) [reply src/App.js:15]: ok cool.

File: src/components/filter/Filter.jsx (hunk 2 above)
Comment (ianuragsharma) [reply src/components/filter/Filter.jsx:135]: Will try to do that. Thanks for the review

**Pull request merged**